android/pytorch_android/src/main/java/com/facebook/pytorch/PytorchScriptModule.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.pytorch;
+
+import com.facebook.jni.HybridData;
+
+import java.nio.Buffer;
+
+public class PytorchScriptModule {
+
+  private NativePeer mNativePeer;
+
+  public PytorchScriptModule(final String modelAbsolutePath) {
```

Comment:
can we follow the same API contract as C++ whenever possible? I.e. have a separate static function for loading the Module?